# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gettext/gettext-no_long_long.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gettext-0.17-orig/gettext-runtime/libasprintf/configure.ac gettext-0.17/gettext-runtime/libasprintf/configure.ac
--- gettext-0.17-orig/gettext-runtime/libasprintf/configure.ac	2008-04-10 18:29:40.000000000 +0000
+++ gettext-0.17/gettext-runtime/libasprintf/configure.ac	2008-04-10 18:30:17.000000000 +0000
@@ -54,7 +54,6 @@
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_INLINE
 AC_TYPE_SIZE_T
-gl_AC_TYPE_LONG_LONG
 gt_TYPE_LONGDOUBLE
 gt_TYPE_WCHAR_T
 gt_TYPE_WINT_T
